apps/web/components/App.tsx

```diff
@@ -55,6 +57,11 @@ export default function App({
 }) {
   const { t } = useLocale();
 
+  const { isSuccess, isLoading, data } = trpc.useQuery(["viewer.integrations"]);
+  const appCredentials: { credentialIds: number[] | undefined } | undefined = data?.other?.items.find(
```

Comment:
I noticed this only searches for credentials for anything under "other" and not the other categories. I'm working on a fix for this right now

Reply:
Oh yeah, you are right there. We should iterate on all kinds before jumping on to this.

Reply:
We may want to handle this from the server. Make an endpoint that specifically returns if an app is installed or not.

Reply:
I've updated the code, and now it should use a endpoint